epoll.c

```diff
@@ -474,7 +478,9 @@ epoll_dispatch(struct event_base *base, struct timeval *tv)
 			if (tv->tv_sec == 0 && tv->tv_usec == 0) {
 				/* we need to exit immediately; timerfd can't
 				 * do that. */
+#if !defined(EVENT__HAVE_EPOLL_PWAIT2)
```

Comment:
Actually right now there is timerfd that is used for more precise timers, in case of epoll_pwait2() it is not required anymore, so avoid using it in case of epoll_pwait2().

Reply:
OK